.eslintrc.js

```diff
@@ -37,6 +37,25 @@ module.exports = {
         'import/no-extraneous-dependencies': 'off',
         'function-paren-newline': 'off',
         'implicit-arrow-linebreak': 'off',
+        'import/order': [
```

Comment:
Nice--thanks for this!

However this should be moved inside the `rules` section. This overrides section only applies to test files.

When I move this to `rules`, I get import errors in some other files also, so feel free to add this rule in a separate patch and fix those errors. (Also I learned that it groups `react-router-dom` with `external` rather than `builtin`, so it says it should come after `CssBaseline` due to alphabetic ordering.)

Reply:
maybe something is wrong with my setup, cos I dont get any issues when I run `npm run lint:fix` locally, all the errors show on ci level.